test/component/summary/testDeceasedSection.js

```diff
@@ -70,11 +77,11 @@ describe('summary-deceased-section', () => {
                         married: deceasedContent.married.question.replace('{deceasedName}', deceasedName),
                         dob: deceasedContent.dob.question,
                         dod: deceasedContent.dod.question,
-                        address: deceasedContent.address.question,
+                        address: deceasedData.deceased.address.formattedAddress,
```

Comment:
This section of the test needs further investigation as it validates the questions being asked bu the answers seem to be missing.....